libopenage/renderer/renderer.h

```diff
@@ -106,7 +117,7 @@ class Renderer {
 	virtual resources::Texture2dData display_into_data() = 0;
 
 	/// Executes a render pass.
-	virtual void render(RenderPass const&) = 0;
+	virtual void render(RenderPass&) = 0;
```

Comment:
Could you please also update the documentation to note that a renderer implementation might modify the render pass?